src/components/dashboard/containers/Receipt/ReceiptTemplates.js

```diff
@@ -0,0 +1,494 @@
+import React, { Fragment } from 'react'

```

Comment:
Can you make it in 2 separate files, and styles in another one.

do not use `formatDateTime`, make new function 'formatDate' for date only